### ISSUE TITLE
Get svelte components working properly

### ISSUE THREAD

**Issue by jt196** [josh-collinsworth/sveltekit-blog-starter#34]
Hey @josh-collinsworth been trying to get components working properly in the markdown posts. 

Currently they're able to import the HTML, but the CSS and JS elements are broken, or just plain don't import. 

I've played around with the config - I believe the `mdsvex()` needs to be preprocessed, but if I do that, it breaks pretty much everything. The blog index, the post page etc. 

Any ideas/thoughts on how to get this working without breaking the blog?

**Comment by josh-collinsworth** [comment]
Hi @jt196 - I created a branch here where I tried creating a basic component called `ClickMe` that toggles some text using JS, and is styled using CSS. I put it into the "heading links examples" Markdown file, and from what I can tell, it's working as it should.

Maybe you can compare with that branch and see if anything might be different from your own implementation? Here's the branch: https://github.com/josh-collinsworth/sveltekit-blog-starter/tree/mdsvex_components

If you aren't able to find anything, sharing a link to your own setup might help.

**Comment by jt196** [comment]
Cheers Josh, I'll give it a go. The component on it's own isn't working but I know I did tweak your set up a bit.

**Comment by jt196** [comment]
So your cloned branch works. I played around a bit with the svelte config file and package json file, neither of them fixed my set up, so I just started afresh and copied my CSS, md, components across and it's all working now. Haven't the energy or time to debug the reasons why it broke. Likely some dumb reason or dependency issue. Thanks for the branch anyway - and the blog share too, it's been very useful with my SK study.
